web-registry/src/components/organisms/ProjectTopSection.tsx

```diff
@@ -280,60 +281,36 @@ function ProjectTopSection({
                     label="credit class"
                     name={creditClassVersion.name}
                     url={
-                      creditClassVersion.metadata?.['http://schema.org/url']?.[
-                        '@value'
-                      ]
+                      isVcsProject
+                        ? `/credit-classes/${creditClass?.onChainId}`
+                        : creditClassVersion.metadata?.[
+                            'http://schema.org/url'
+                          ]?.['@value']
                     }
```

Comment:
@blushi Linking to new ledger credit class page now. But still linking to carbonplus-grasslands if not VCS. Not sure if this is the best logic.

Reply:
I believe that'll work for now